jedi.inc

```diff
@@ -168,6 +168,8 @@
   DELPHIX_SYDNEY         Alias for DELPHI27
   DELPHI28               Defined when compiling with Delphi 11 for Win32 (Codename OLYMPUS)
   DELPHIX_ALEXANDRIA     Alias for DELPHI28
+  DELPHI29               Defined when compiling with Delphi 12 for Win32 (Codename YUKON)
+  DELPHIX_ATHENS         Alias for DELPHI29
```

Comment:
Actually, there should not be an X in there because the X was for "10" back when Embarcadero decided not to increase the major version anymore (10.1, 10.2...)
Now that this nonsense has been stopped, the X should not be used any more so this should really be `DELPHI_ATHENS`

I know that this was not noticed with ALEXANDRIA but at the time, it wasn't clear whether would be named 11 or 10.5
Now that `DELPHIX_ALEXANDRIA` is out, we must not revert it, but for a new value, the X must be removed.

Reply:
Fixed. Thank you!